governance/polls/MIP10c14-SP7: Appoint DeFi Saver as Oracle Light Feed.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Appoint DeFi Saver as Oracle Light Feed (MIP10c14-SP7) - April 5, 2021
+summary: Signal your support or opposition for appointing DeFi Saver as Oracle Light Feed
+discussion_link: https://forum.makerdao.com/t/mip10c14-sp7-proposal-appoint-defi-saver-as-a-light-feed/7207
+vote_type: Plurality Voting
+categories:
+   - Oracles
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Appoint DeFi Saver as Oracle Light Feed (MIP10c14-SP7) - April 5, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of DeFi Saver. This Governance Poll will be active for three days beginning on Monday, April 5 at 16:00 UTC.
```

Comment:
Thinking the Poll is probably being placed on behalf of the Oracles Team?

Reply:
I mean, the entity proposing it is technically DeFi Saver. Let's defer to @LongForWisdom.

Reply:
The poll is going up on behalf of the Oracles team, Nik has to approve the submissions for voting.